### PR TITLE
Learning about classes

### PR DESCRIPTION
**Pull request by atorrico09**
### Tasks:

- classes
- functions
- created Todo class

### CODE REVIEW

File: Todo.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    def __init__(self, name, owner, is_important=False, is_done=False):
+        self.name = name
+        self.owner = owner
+        self.is_important = is_important
+        self.is_done = is_done
+
+    def toggle_is_important(self):
+        self.is_important = not self.is_important
+
+    def toggle_is_done(self):
```
Comment (atorrico09) [inline Todo.py:11]: Could you try rename according our conventions? e.g. `ToggleIsDone`

Comment (ilikecoding17) [reply Todo.py:11]: done, thanks

**Review by ilikecoding17 — CHANGES_REQUESTED** [review]
please make it better

**Review by ilikecoding17 — APPROVED** [review]
wow not terrible

**Pull request merged**